domains/midair.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zMidair",
+        "email": "midair4546@gmail.com"
+    },
+    "record": {
+        "URL": "midair.is-a.dev"
```

Comment:
invalid URL record, also you cant use your is-a.dev domain which you want in any of these records

Reply:
Wait, one sec, I will edit mb

Reply:
Can you edit the record value to “midair.ssite.link”? Idk how I can do that

Reply:
Updated